### ISSUE TITLE
show federation-id in sharing details if file/folder is shared from another user

### ISSUE THREAD

**Issue by devnull4242** [nextcloud/server#24180]
The sharing details show me with whom I have shared files and directories. But it does not show me who shared this folder or file with me. I can only see the federation-id on mouse-over at the file or directory listing. 

Perhaps it is possible also to show the federation-id of the user who shared the file or folder with me. Please in that way that i can copy and paste the fedaration-id.

Read for more details:
https://help.nextcloud.com/t/extract-cloud-federated-id-from-a-share/86245

Example:
![example](https://user-images.githubusercontent.com/58322182/99386195-5e950600-28d2-11eb-9269-007a64e3cf4e.png)

**Comment by szaimen** [comment]
cc @nextcloud/designers does this make sense and is it feasible?

**Comment by juliushaertl** [comment]
Makes sense to unify this to make it clear where a file is coming from also for federated shares.

Currently for local shares:

![image](https://user-images.githubusercontent.com/3404133/127814018-647f4499-98db-48f8-8fb8-de62012bdbe9.png)

Remote shares:
![image](https://user-images.githubusercontent.com/3404133/127814082-a276d1c1-551a-4e3b-9127-c0050f4a6ace.png)
